src/main/java/dev/openfeature/sdk/OpenFeatureAPI.java

```diff
@@ -18,9 +20,10 @@ public class OpenFeatureAPI {
     static AutoCloseableReentrantReadWriteLock hooksLock = new AutoCloseableReentrantReadWriteLock();
     static AutoCloseableReentrantReadWriteLock providerLock = new AutoCloseableReentrantReadWriteLock();
     static AutoCloseableReentrantReadWriteLock contextLock = new AutoCloseableReentrantReadWriteLock();
-    private FeatureProvider provider;
     private EvaluationContext evaluationContext;
     private List<Hook> apiHooks;
+    private Map<String, FeatureProvider> providers = new ConcurrentHashMap<>();
+    private static String DEFAULT_PROVIDER_KEY = "very-secret-string-which-you-shouldnt-use";
```

Comment:
shouldn't be`DEFAULT_PROVIDER_CLIENT_NAME_KEY`? we discussed lot about this in ofep. https://github.com/open-feature/ofep/pull/56#discussion_r1142280472

Reply:
There is no client name for the default provider. The default provider is the default if no client name is specified.